piptools/repositories/pypi.py

```diff
@@ -284,10 +284,10 @@ def get_dependencies(self, ireq):
                         os.environ["PIP_REQ_TRACKER"] = prev_tracker
                     else:
                         del os.environ["PIP_REQ_TRACKER"]
-                try:
-                    self.wheel_cache.cleanup()
-                except AttributeError:
-                    pass
+
+                # WheelCache.cleanup() introduced in pip==10.0.0
+                if PIP_VERSION >= (10,):
```

Comment:
Does pip-tools actually support versions lower than pip v10?

Reply:
pip-tools supports 8.1.1, 9.0.1, and 9.0.3.

Reply:
![tenor](https://user-images.githubusercontent.com/1772890/67630577-386a8580-f8cd-11e9-9c04-651315236c31.gif)

Reply:
Wouldn't it make sense to simply up the required pip version? Pip will complain anyways, if it's not up to date, so most people will have it on a newer version and even if we don't if pip-tools sets the correct requirement, it will simply update pip.

Reply:
Right, I have the feeling I am missing something fundamental 🤔

Reply:
He-he 😊 You should join the discussion #926!

Reply:
haha, yes I should :P